src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -104,6 +110,9 @@ public class BlobFileStore : IFileStore
                 }
 
                 BlobContentInfo info = await blobClient.CommitBlockListAsync(blockLengths.Keys, cancellationToken: cancellationToken);
+
+                EmitTelemetry("StoreFileInBlocksAsync", OperationType.Input, stream.Length);
```

Comment:
Are we sure this stream's length can always be read? This is not a required property for all streams, like for ones whose length is not known ahead of time. I think for us this is always the case as we buffer, but please double-check. We can always compute it above too

Reply:
Although, I see us accessing it in other parts of this file, so this comment may be moot